Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN apt-get update \
 
 COPY ./src src/
 
-CMD ["streamlit", "run", "src/app.py"]
+CMD ["gunicorn", "-b 0.0.0.0:80", "src.app:server"]
```

Comment:
that would be better to have the default python app running command (gunicorn would be the docker-compose command: the reason behind this is that gunicorn can be replaced with uvicorn or other tools)